README.md

```diff
@@ -303,6 +306,21 @@ Remove the line to DROP FOREIGN KEY.
 
 Composite foreign key may not work for now.
 
+### CREATE (OR REPLACE) VIEW
+
+```diff
+ CREATE OR REPLACE VIEW foo AS
+   select u.id as id, p.id as post_id
+   from  (
+     mysqldef_test.users as u
+     join mysqldef_test.posts as p on ((u.id = p.user_id))
```

Comment:
Is that intentional that a view definition needs to include a database name, or is that a compromise for making the current implementation work? Not complaining about that in this PR, but I'd like to check what you'd actually like for future changes.

Reply:
The result of [this query](https://github.com/k0kubun/sqldef/blob/e94187cf27/adapter/mysql/mysql.go#L60) it is included a database name.

Reply:
Yes, I know it, and that's not what I'm asking. "Is that a compromise for making the current implementation (using `show full tables`) work?"

Reply:
I'm sorry, I'm not good at English. The answer is "yes".